modules/openapi-generator/src/main/resources/Java/libraries/okhttp-gson/ApiClient.mustache

```diff
@@ -139,16 +140,25 @@ public class ApiClient {
     {{/-first}}
     {{/oauthMethods}}
     {{/hasOAuthMethods}}
-    private void init() {
+    private void initHttpClient() {
+        initHttpClient(Collections.<Interceptor>emptyList());
+    }
+
+    private void initHttpClient(List<Interceptor> interceptors) {
         OkHttpClient.Builder builder = new OkHttpClient.Builder();
         builder.addNetworkInterceptor(getProgressInterceptor());
-        httpClient = builder.build();
-
+        for (Interceptor interceptor: interceptors) {
+            builder.addInterceptor(interceptor);
+        }
         {{#useGzipFeature}}
-        // Enable gzip request compression
-        httpClient.interceptors().add(new GzipRequestInterceptor());
+            // Enable gzip request compression
+            builder.addInterceptor(new GzipRequestInterceptor());
```

Comment:
@trohrberg seems like these 2 lines are not indented correctly. I'll file a separate PR to update it after merging this one.